test/interop/fortran/genFortranInterface/chapelProcs.chpl

```diff
@@ -0,0 +1,30 @@
+var chplInt: int;
+var chplReal: real;
+
+// Note: These exported functions need to be all lowercase
```

Comment:
whoah? If that's important for users, maybe we need to warn if the fortran interface tries to use upper case functions?

Reply:
I think I was wrong in that comment. At least gfortran doesn't seem to care about the capitalization.